extensions/azure-extensions/src/main/java/io/druid/storage/azure/AzureLoadSpec.java

```diff
@@ -31,24 +31,24 @@
 public class AzureLoadSpec implements LoadSpec
 {
   @JsonProperty
-  private final String storageDir;
+  private final String blobPath;
 
   private final AzureDataSegmentPuller puller;
 
   @JsonCreator
   public AzureLoadSpec(
-      @JacksonInject AzureDataSegmentPuller puller,
-      @JsonProperty("storageDir") String storageDir
```

Comment:
The official docs list 3 parts `http://<storage account>.blob.core.windows.net/<container>/<blob>`

If we were to map this directly to S3 then storage account would be the bucket (maybe?), and container/blob would be the key.  If that is true, then we should probably have each of the three parts explicitly in the loadspec.